### PR TITLE
made zommable map functionality

### PR DESCRIPTION
**Pull request by bbassett**

### CODE REVIEW

File: blocks/el/map/features.jade [hunk 1]
```diff
@@ -18,4 +18,29 @@ export.
...
+  }
+
+export.
+  function componentWillUnmount() {
+    this.getDOMNode().removeEventListener('zoom');
+  }
+
+export.
+  function componentDidMount() {
+    if (!process.env.FEATURE_MAP_ZOOM) return
```
Comment (bbassett) [inline blocks/el/map/features.jade]: it's all behind a feature flag

File: blocks/el/map/features.jade [hunk 2]
```diff
@@ -18,4 +18,34 @@ export.
...
+    var svg = self.svg = d3.select(this.getDOMNode())
+    self.paths = svg.selectAll('path')
+
+    var minScale = 0.75;
+    var maxScale = 12;
+
+    var zoom = d3.behavior.zoom()
+      .scaleExtent([minScale, maxScale])
+      .on('zoom', function(){
+        self.paths.attr("transform", "translate(" + d3.event.translate.join(",") + ")scale(" + d3.event.scale + ")" );
```
Comment (mndvns) [inline blocks/el/map/features.jade]: double quotes

File: blocks/el/map/features.jade (hunk 1 above)
Comment (mndvns) [inline blocks/el/map/features.jade]: don't forget semicolons
`if (!process.env.FEATURE_MAP_ZOOM) return;`

File: blocks/el/map/features.jade [hunk 3]
```diff
@@ -18,4 +18,34 @@ export.
...
+    this.getDOMNode().removeEventListener('zoom');
+  }
+
+export.
+  function componentDidMount() {
+    if (!process.env.FEATURE_MAP_ZOOM) return
+    var self = this;
+
+    var svg = self.svg = d3.select(this.getDOMNode())
+    self.paths = svg.selectAll('path')
```
Comment (mndvns) [inline blocks/el/map/features.jade]: semicolons on `33` and `34`

**Pull request merged**